python/ray/data/read_api.py

```diff
@@ -698,6 +704,115 @@ def read_parquet(
     )
 
 
+@PublicAPI
+def read_delta(
+    uri: str,
+    delta_package: str = None,
```

Comment:
is this a required parameter? either remove `= None`, or put it below * as `Optional[str]`.